framework/include/utils/SerializerGuard.h

```diff
@@ -0,0 +1,27 @@
+#ifndef SERIALIZERGUARD_H
```

Comment:
Header

Reply:
Do we need to copy those around?  I always just figure that @permcody will get them with his script every now and again

Reply:
Well I can, but there's no reason not to add them on new files as they go in.